pubpub/platform
# Slug editing for PubFields on create

## Issue(s) Resolved
Closes https://github.com/pubpub/v7/issues/407

## Test Plan

1. Visit i.e. http://localhost:3000/c/croccroc/fields
2. Click the "New Field" button
3. You should be able to add a slug here. The slug will autogenerate based on the title, but can be edited after the title (but if you change the title, it'll try to autogenerate again. we can change that behavior if we want)
4. Editing an existing field will not show the slug field (although maybe it should, just disabled?)

## Screenshots (if applicable)

https://github.com/user-attachments/assets/7ea2a728-9f15-4e8c-81b8-031e2069d22a

![image](https://github.com/user-attachments/assets/d140c81d-6060-4e1a-b2a5-b50425035ec9)

## Optional

### Notes/Context/Gotchas

I made this without a real design, so feel free to say it's bad 😄 

### Supporting Docs
